lib/rubygems/gemcutter_utilities.rb

```diff
@@ -132,12 +133,26 @@ def sign_in(sign_in_host = nil)
 
     email = ask "   Email: "
     password = ask_for_password "Password: "
+
+    hostname = Socket.gethostname
+    default_key_name = "#{hostname}"
+    key_name = ask "Key name [#{default_key_name}]: "
```

Comment:
I think the suggested key name should instead be a safe version of the host that the user is signing into. Ie: `rubygems_org`

Reply:
In my mind, the purpose of the name of the key is to identify where the key is being used in the client. I am sorry but I don't understand how naming the key `rubysgems_org` would help in this or perhaps serve a greater purpose? please elaborate.